package.json

```diff
@@ -79,6 +81,13 @@
   },
   "browser": {
     "crypto": false,
-    "./lib/sha256digest.js": "./lib/sha256digest-browser.js"
+    "./lib/sha256digest.js": "./lib/sha256digest-browser.js",
+    "fast-text-encoding": false,
+    "isomorphic-webcrypto": false
```

Comment:
@davidlehn -- is this and the `crypto: false` required here? Don't the aliases suffice since code paths must not require any of these other libs anyway?

Reply:
I'm not sure.. I think some bundlers require from the package.json and not code paths? I can remove em though.